# Contains the markers that are found on a link

# Header specifying link
Header header

# Aruco markers on the link
ArucoGridboardWithPose[] markers
